src/Components/benchmarkapps/Wasm.Performance/Driver/Program.cs

```diff
@@ -0,0 +1,263 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.ExceptionServices;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Hosting.Server;
+using Microsoft.AspNetCore.Hosting.Server.Features;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Remote;
+using OpenQA.Selenium.Support.UI;
+using Wasm.Performance.Driver;
+using DevHostServerProgram = Microsoft.AspNetCore.Blazor.DevServer.Server.Program;
+
+namespace Wasm.Performance.Driver
+{
+    public partial class Program
```

Comment:
Out of interest, does this code come from somewhere else, or is it newly-created for this PR?

I'm asking because it seems to contain a bunch of comments about working around issues and it's unusual to see that in the v1 of any particular bit of code.

Reply:
It makes me wonder if we're duplicating knowledge across projects through copy-paste and if there are opportunities to centralise this infrastructure somehow.